src/trace/ProxyTracer.ts

```diff
@@ -38,6 +38,11 @@ export class ProxyTracer implements Tracer {
     return this._getTracer().startSpan(name, options, context);
   }
 
+  startActiveSpan<F extends (span: Span) => ReturnType<F>>(): ReturnType<F> {
```

Comment:
Your sinon stub is broken because it can't inspect the arguments of this type I believe.

```suggestion
  startActiveSpan<F extends (span: Span) => unknown>(
    _name: string,
    _options: F | SpanOptions,
    _context?: F | Context,
    _fn?: F
  ): ReturnType<F> {
```